docs/prev/2022/program.md

```diff
@@ -161,3 +161,5 @@ We plan to have a combination of Invited Speakers, Research Paper Presentations,
     </tr>
 </tbody>
 </table>
+
+## [Proceedings](https://ieeexplore.ieee.org/xpl/conhome/10029390/proceeding) from CANOPIE-HPC SC22
```

Comment:
This edition should already have a link to proceedings published in the IEEE CS Digital Library, see https://github.com/supercontainers/canopie-hpc/blob/983fbb62d668fdf2e534aac0d82510f4a454adf9/docs/prev/2022/program.md?plain=1#L10

Maybe this could be refactored to display both links/portals?

Reply:
Ah, Shane just updated the page to add that proceedings link, that's why I did not find it in the diff!